lib/rubocop/cop/commissioner.rb

```diff
@@ -64,6 +63,11 @@ def reset_errors
       end
 
       # TODO: Bad design.
+      def remove_irrelevant_cops(filename)
+        @cops.select! { |cop| cop.relevant_file?(filename) }
```

Comment:
This method name and the implementation kinda points in opposite directions. I thought of either naming the method `#select_relevant_cops` or changing the implementation to `@cops.reject! { |cop| !cop.relevant_file?(filename) }`. What do you think?